hub/ui/src/components/search-bar/SearchBar.tsx

```diff
@@ -36,8 +36,11 @@ const SearchBar: React.FC = (props: any) => {
   const onTextChanged = (e: any) => {
     const value = e;
     let suggestions: any = [];
-    const regex = new RegExp(`${value}`, 'i');
-    suggestions = props.TaskDataList.sort().filter((v: any) => regex.test(v.name));
+    const nameregex = /-/gi;
+    const regex = new RegExp(`${ value.replace(nameregex, ' ') }`, 'i');
+    suggestions = props.TaskDataList.sort().filter((v: any) =>
+      regex.test(v.displayName.replace(nameregex, ' ')) ||
```

Comment:
I think it does not search if the display name correctly

For e.g `Upload to GCS`

Reply:
it's searching correctly check once